src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,252 @@
+# Motion
+
+Support for Motion was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+Several [example Motion datasets](https://github.com/bids-standard/bids-examples)
+have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+## Motion recording data
+
+Template:
+
+└─ sub-<label>\
```

Comment:
need formatting to follow other similar examples, e.g.

```shell
$> git grep -A7 'Template:'
...
src/03-modality-agnostic-files.md:Template:
src/03-modality-agnostic-files.md-
src/03-modality-agnostic-files.md-```Text
src/03-modality-agnostic-files.md-sub-<label>/
src/03-modality-agnostic-files.md-    [ses-<label>/]
src/03-modality-agnostic-files.md-        sub-<label>[_ses-<label>]_scans.tsv
src/03-modality-agnostic-files.md-        sub-<label>[_ses-<label>]_scans.json
src/03-modality-agnostic-files.md-```
```

Reply:
Hopefully taken care of now :)